sources/targets/Stride.Core.props

```diff
@@ -60,12 +58,12 @@
   </PropertyGroup>
 
   <PropertyGroup Condition=" '$(StrideRuntime)' == 'true' ">
-    <!-- Add net6.0 no matter what (needed for references) -->
-    <StrideRuntimeTargetFrameworks>net6.0</StrideRuntimeTargetFrameworks>
-    <StrideRuntimeTargetFrameworks Condition="$(_StridePlatforms.Contains(';Windows;')) And '$(StrideRuntimeWindowsNet6)' == 'true'">$(StrideRuntimeTargetFrameworks);net6.0-windows7.0</StrideRuntimeTargetFrameworks>
+    <!-- Add net8.0 no matter what (needed for references) -->
+    <StrideRuntimeTargetFrameworks>net8.0</StrideRuntimeTargetFrameworks>
+    <StrideRuntimeTargetFrameworks Condition="$(_StridePlatforms.Contains(';Windows;')) And '$(StrideRuntimeWindowsNet6)' == 'true'">$(StrideRuntimeTargetFrameworks);net8.0-windows7.0</StrideRuntimeTargetFrameworks>
```

Comment:
Shouldn't `$(StrideRuntimeWindowsNet6)` be renamed to `$(StrideRuntimeWindowsNet8)`?

Reply:
good point, maybe even `StrideRuntimeWindowsNetCore`?